thorlcr/graph/graph_lcr.cmake

```diff
@@ -54,6 +54,7 @@ include_directories (
          ./../../common/thorhelper 
          ./../../roxie/roxiemem
          ./../../system/security/shared
+         ./../../system/tbb_sm/tbb/include
```

Comment:
This is not needed here - its needed in common/thorhelper/CMakeLists.txt to replace the ${TBB_INCLUDE_DIR}